test/test-functions

```diff
@@ -3036,6 +3036,7 @@ _test_cleanup() {
         fi
         [[ -n "$TESTDIR" ]] && rm -vfr "$TESTDIR"
         [[ -n "$STATEFILE" ]] && rm -vf "$STATEFILE"
+        [[ -n "$STATEDIR" ]] && rm -vfr "$STATEDIR"
```

Comment:
This always removes TESTLOG, even on failure, so it cannot be consulted anymore. I'll change it to cleanup this only on success.

Reply:
https://github.com/systemd/systemd/pull/27045

Reply:
Indeed, thanks for the follow-up.